<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SolverOPD</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/global-network.png</normaloff>icons/global-network.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 211, 105);</string>
    </property>
    <property name="text">
     <string>Solve</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFITS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>421</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>421</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>FITS File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFile">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 211, 105);</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>503</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelPlateSolve">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>451</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>TARGET</string>
    </property>
    <widget class="QLineEdit" name="txtDEC">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>60</y>
       <width>191</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>+00 00 00.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRA">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>191</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>00 00 00.0</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinFOV">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="suffix">
      <string>'</string>
     </property>
     <property name="value">
      <double>11.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>FOV (arcmin)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Right Ascension</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>431</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Coordinates need to be in J2000 format</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Declination</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinStars">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Plot n Stars</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>451</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>RESULT</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Center DEC</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Center RA</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDECSolved">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>+00 00 00.0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRASolved">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>00 00 00.0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>10</y>
      <width>550</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126, 126, 126);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>icons/img.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>451</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Engravers MT</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(212, 212, 105);</string>
    </property>
    <property name="text">
     <string>PICO DOS DIAS OBSERVATORY</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
